### ISSUE TITLE
Warning: overwriting garbling

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#297]
v2.24.3
```
?- [user].
'\n' :- write('\n').
   true.
?- [user].
'\n' :- write('\n').
Warning: overwriting 
/0
   true.

?- [user].
'\r' :- write('\r').
   true.
?- [user].
'\r' :- write('\r').
/0rning: overwriting  % garbled
   true.
?- 
```
Overwriting /0 ?

**Comment by infradig** [comment]
You're right, quotes should be on.

**Comment by flexoron** [comment]
Fixed with v2.24.4 but don't know why /0 is displayed:
```
?- [user].
'/a'.
   true.
?- [user].
'/a'.
Warning: overwriting '/a'/0
   true.
?- 
```

**Comment by infradig** [comment]
That's the arity. An atom has zero arity.
